scripts/installscripts/buster-install-default.sh

```diff
@@ -820,9 +867,9 @@ install_main() {
     sudo lighttpd-enable-mod fastcgi-php
     sudo service lighttpd force-reload
 
-    # create copy of GPIO script
-    sudo cp "${jukebox_dir}"/misc/sampleconfigs/gpio-buttons.py.sample "${jukebox_dir}"/scripts/gpio-buttons.py
-    sudo chmod +x "${jukebox_dir}"/scripts/gpio-buttons.py
+#Not needed anymore    # create copy of GPIO script
```

Comment:
You deleted this files, so you could just delete these commands as well. It’s versioned, so there shouldn’t be a problem.

Reply:
Yes you are right! I did the commenting out at the beginning and forgot about it.